src/axom/inlet/Inlet.hpp

```diff
@@ -194,80 +194,45 @@ class Inlet : public SchemaCreator
   //
 
   /*!
-   *****************************************************************************
-   * \brief Gets a Boolean value out of the Datastore.
-   *
-   * Retrieves the Field value out of the DataStore.  This Field may not have
-   * been actually present in the input file and will be indicted by the return
-   * value. 
-   *
-   * \param [in] name Name of the Field value to be gotten
-   * \param [out] value Value to be filled
-   *
-   * \return True if the value was found in the Datastore
-   *****************************************************************************
-   */
-  bool get(const std::string& name, bool& value)
-  {
-    return m_globalTable->get(name, value);
-  }
-
-  /*!
-   *****************************************************************************
-   * \brief Gets a Double value out of the Datastore.
-   *
-   * Retrieves the Field value out of the DataStore.  This Field may not have
-   * been actually present in the input file and will be indicted by the return
-   * value. 
-   *
-   * \param [in] name Name of the Field value to be gotten
-   * \param [out] value Value to be filled
-   *
-   * \return True if the value was found in the Datastore
-   *****************************************************************************
-   */
-  bool get(const std::string& name, double& value)
+   *******************************************************************************
+   * \brief Gets a value of arbitrary type out of the datastore
+   * 
+   * Retrieves a value of user-defined type, i.e., not double, int, bool, or string.
+   * 
+   * \param [in] name The name of the subtable representing the root of the object
+   * \return The retrieved value
+   * \tparam The type to retrieve
+   * \pre Requires a specialization of from_inlet<T>(axom::inlet::Table&)
+   * \note This function does not indicate failure
```

Comment:
>\note This function does not indicate failure

Is this still true?  Doesn't it call `SLIC_ERROR` on (some) errors?

Reply:
I've added a clarification to this note - failure is not indicated in a way that can be handled by a program as neither a boolean return value or exceptions are used.